go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/massdriver-cloud/mass
 
 go 1.22.0
 
+replace github.com/osteele/liquid => github.com/massdriver-cloud/liquid v1.4.0
```

Comment:
Using replace here as it makes the fork cleaner (don't have to modify the go.mod of the fork).